feature-libs/product-configurator/rulebased/components/price-async/configurator-price-async.component.spec.ts

```diff
@@ -0,0 +1,314 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { I18nTestingModule } from '@spartacus/core';
+import {
+  CommonConfigurator,
+  ConfiguratorRouterExtractorService,
+} from '@spartacus/product-configurator/common';
+import { DirectionMode, DirectionService } from '@spartacus/storefront';
+import { EMPTY, Observable, Subject, of } from 'rxjs';
+import { CommonConfiguratorTestUtilsService } from '../../../common/testing/common-configurator-test-utils.service';
+import { ConfiguratorCommonsService } from '../../core';
+import { Configurator } from '../../core/model/configurator.model';
+import { ConfiguratorTestUtils } from '../../testing/configurator-test-utils';
+import {
+  ConfiguratorPriceAsyncComponent,
+  ConfiguratorPriceAsyncComponentOptions,
+} from './configurator-price-async.component';
+
+/**
+ * group1@attribute_1_1
+ * - value_1_1: -$100
+ * - value_1_2: $0
+ * - value_1_3: $200
+ *
+ * group1@attribute_1_2
+ * - value_1_1: -$100
+ * - value_1_2: $100
+ * - value_1_3: $300
+ */
+const mockConfigTemplate: Configurator.Configuration = {
+  ...ConfiguratorTestUtils.createConfiguration('c123'),
+  pricingEnabled: true,
+  priceSupplements: ConfiguratorTestUtils.createListOfAttributeSupplements(
+    false,
+    1,
+    0,
+    2,
+    3
+  ),
+};
+
+const price1: Configurator.PriceDetails = {
+  value: 10,
+  formattedValue: '$10.00',
+  currencyIso: 'USD',
+};
+
+let direction = DirectionMode.LTR;
```

Comment:
@Uli-Tiger, why is the direction relevant?

Reply:
There a re some tests with LTR and some with RTL.  ($100- vs -$100)